Foundation/NSFileManager.swift

```diff
@@ -343,7 +343,19 @@ open class FileManager : NSObject {
         This method replaces fileSystemAttributesAtPath:.
      */
     open func attributesOfFileSystem(forPath path: String) throws -> [FileAttributeKey : Any] {
-        NSUnimplemented()
+        var s = statfs()
+        guard statfs(path, &s) == 0 else {
+            throw _NSErrorWithErrno(errno, reading: true, path: path)
+        }
+        
+        var result = [FileAttributeKey : Any]()
+        let blockSize = UInt64(s.f_bsize)
+        result[.systemSize] = NSNumber(value: blockSize * s.f_blocks)
+        result[.systemFreeSize] = NSNumber(value: blockSize * s.f_bavail)
+        result[.systemNodes] = NSNumber(value: s.f_files)
+        result[.systemFreeNodes] = NSNumber(value: s.f_ffree)
```

Comment:
Also one other note: since swift-corelibs-foundation does not have bridging like the objc version- by storing the NSNumber here you are requiring callers to as cast to NSNumber and things like UInt64 (which they probably would want) will not work.